src/resolvers.ts

```diff
@@ -53,5 +53,40 @@ export const resolvers: ResolverMap = {
       console.warn(`Could not load value ${argument} from credstash: ${error}`);
       return undefined;
     });
+  },
+  aws: async (argument: string) => {
+    const params = argument.split('.');
+    const [secretKey, additonalValue] = params;
```

Comment:
I would rename secretKey to secretId so that it doesn’t sound like a “key”.  I misread this initially thinking it was a key.